app/formSchema/validate.ts

```diff
@@ -4,9 +4,35 @@ import schema from './schema';
 const validate = (formData) => {
   const errorSchema = validateFormData(formData, schema)?.errorSchema;
 
+  const fileUploadPages = [
+    'templateUploads',
+    'supportingDocuments',
+    'coverage',
+  ];
+
+  // Remove 'should be string' errors for file upload fields which are saved as an array
+  Object.keys(errorSchema).forEach((formPage) => {
```

Comment:
Do we need this part? I commented it out locally and the bug isn't being reproduced

Reply:
Yes it is needed, try commenting that out and running the `ApplicationForm` test. Without it the validation will return those pages with empty objects causing the review page checkbox and alert section to show incorrectly.

If you want to test this past the tests you can see it by importing the `mockFormData` and running that through the validate function.

Reply:
Ah gotcha!